src/lib/components/Card/Card.tsx

```diff
@@ -38,7 +38,7 @@ export type CardProps =
       | {
           /** Allows to provide a custom render function for the image component. Useful in Next.JS and Gatsby. **Setting this will disable `imgSrc` and `imgAlt`**.
            */
-          renderImage?: () => JSX.Element;
+          renderImage?: (config: { theme: DeepPartial<FlowbiteCardTheme>; horizontal: boolean }) => JSX.Element;
```

Comment:
I don't think that we need this `config` object. Just passing the two parameters is good enough, otherwise, if you want to use the config object, you need to create an interface for it.

Reply:
I am not sure that I understand your point correctly. Are you saying: "In order to get proper types in the storybook docs you need to create a custom interface here."? That does not help, I fear since the types for the theme in the card props have the same poor quality:
![Screenshot 2023-05-15 at 07 22 50](https://github.com/themesberg/flowbite-react/assets/1150767/edb037f5-f04c-458d-81f2-3989fee2dda0)
I think one would have to somehow define a "Theme" type alias or something for the docs feature of storybook but since this is already "broken", I do not see a point in fixing it here.
If you want me to change the interface for other reasons, I can of course do it. What do you like? This:
```suggestion
          renderImage?: (theme: DeepPartial<FlowbiteCardTheme>, horizontal: boolean) => JSX.Element;
```
?
As I said, I personally find it a bit difficult because one has to remember which position is for which value in the arguments, as opposed to one argument that has named keys. Also adding more values later might become quite tedious for users when we have 4 or 5 arguments here. But I also do not have strong opinions on it (as I will probably never use these arguments in my `renderImage` functions).

Reply:
Please give me feed back @rluders I do not want to drag this on for too long. From experience I can say: The longer a PR stays open, the less likely it will get merged eventually.

Reply:
@levino I got it, but this is not my main job. I have a lot on my plate. But I'll try to test it this week.